### PR TITLE
SQL-1128: Implement signing for connector pqx file

### PR DESCRIPTION
**Pull request by bucaojit**
Added the self-signed certificate to evergreen and that is being used to sign the `MongoDBAtlasODBC.pqx` file.
Added helper scripts to add/remove the thumbprint to the registry. 
Confirmed that Power BI loads the mongodb connector with higher security settings.

### CODE REVIEW

File: connector/MongoDBAtlasODBC.pq [hunk 1]
```diff
@@ -5,7 +5,7 @@ section MongoDBAtlasODBC;
 // This should be set to false before release. Tracing is done through a call to 
 // Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a 
 // no-op and simply returns the original value.
-EnableTraceOutput = true;
+EnableTraceOutput = false;
```
Comment (bucaojit) [inline connector/MongoDBAtlasODBC.pq:8]: Added a drive-by fix to disable tracing now that we will be submitting the connector for certification.

Comment (nbagnard) [reply connector/MongoDBAtlasODBC.pq:8]: ahah, I see, one less JIRA to deal with :)

**Review by nbagnard — APPROVED** [review]
LGTM! Thank you!

**Review by jchemburkar — APPROVED** [review]
LGTM!

**Comment by bucaojit** [comment]
evergreen merge
